tests/integration/issues/github_2295/test_issue.py

```diff
@@ -0,0 +1,59 @@
+import os
+from typing import Iterable, List
+
+import pytest
+
+from jina.drivers.search import KVSearchDriver
+from jina.executors.indexers.keyvalue import BinaryPbIndexer
+from jina.flow import Flow
+from jina import Document, DocumentSet
+
+from tests import validate_callback
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+class SearchDocIndexer(BinaryPbIndexer):
+    def query(self, jina_id: str = None, mongo_ids: List[str] = None):
```

Comment:
Why only 1 `jina_id` but multiple `mongo_ids`?

Reply:
this is provided by the client, is just an example